spec/features/amusement_parks/show_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "amusement parks show page" do
+
+  # User Story 3
+  it 'lists the name, price of admission, and unique mechanics working on the parks rides' do
+    # Setup
+    @six_flags = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @hurler = @six_flags.rides.create!(name: 'The Hurler', thrill_rating: 7, open: true)
+    @scrambler = @six_flags.rides.create!(name: 'The Scrambler', thrill_rating: 4, open: true)
+    @ferris = @six_flags.rides.create!(name: 'Ferris Wheel', thrill_rating: 7, open: false)
+    @dave = Mechanic.create!(name: "Dave Williams", years_experience: 30)
+    @barbara = Mechanic.create!(name: "Barbara Jones", years_experience: 15)
+    @kyle = Mechanic.create!(name: "Kyle Smith", years_experience: 22)
+    MechanicRide.create!(ride_id: @hurler.id, mechanic_id: @dave.id)
+    MechanicRide.create!(ride_id: @scrambler.id, mechanic_id: @dave.id)
+    MechanicRide.create!(ride_id: @ferris.id, mechanic_id: @dave.id)
+    MechanicRide.create!(ride_id: @hurler.id, mechanic_id: @barbara.id)
+    MechanicRide.create!(ride_id: @ferris.id, mechanic_id: @barbara.id)
+    MechanicRide.create!(ride_id: @hurler.id, mechanic_id: @kyle.id)
+
+    # As a visitor, when I visit an amusement park’s show page,
+    visit "/amusement_parks/#{@six_flags.id}"
+
+    # Then I see the name and price of admissions for that amusement park
+    within ".park_info" do
+      expect(page).to have_content("#{@six_flags.name}")
+      expect(page).to have_content("#{@six_flags.admission_cost}")
+    end
```

Comment:
With this test, is there any reason why you have them interpolated if you don't have any text also included in the quotations? What's the difference between `expect(page).to have_content("#{@six_flags.name}")` and `expect(page).to have_content(@six_flags.name})`? 🤔 

If you're going to use string interpolation, you will want to add some text or it defeats the purpose: `expect(page).to have_content("Admission Cost: $#{@six_flags.admission_cost}")`.